LogicalTypes.md

```diff
@@ -59,7 +60,7 @@ Compatibility considerations are mentioned for each annotation in the correspond
 
 ### STRING
 
-`STRING` may only be used to annotate the binary primitive type and indicates
+`STRING` may only be used to annotate the BYTE_ARRAY primitive type and indicates
```

Comment:
It seems the spec is unclear about whether or not `STRING` and `ENUM` can annotate `FIXED_LENGTH_BYTE_ARRAY`. Literally it is reasonable to annotate `FIXED_LENGTH_BYTE_ARRAY`, right? I'm not sure if there is any use case in the wild.

Reply:
I don't know. For `DECIMAL` the spec calls out "byte arrays, binary and fixed" as valid physical types. I'd take the lack of mention of fixed-length here to indicated that only the `BYTE_ARRAY` physical type is allowed. Do any current implementations allow fixed-length strings?

Reply:
I think the existing wording is pretty clear of we take "binary primitive type" to mean BYTE_ARRAY and thus in my opinion this is not a change to the spec. 

Perhaps we could send a note to dev@parquet.apache.org just highlighting this clarification in case someone wants to chime in and say they read it to mean FIXED_LENGTH_BYTE_ARRAY was also supported

Reply:
The sense I'm getting from the M/L is that people are open to adding it, but haven't seen any evidence that any one actually supports adding `STRING` to `FLBA`. IMO we should ship this as is, and start a new thread to gauge support for actually modifying the spec to allow for fixed length strings. Given the variable width of UTF-8 characters, I'd think padding would have to be added to account for up to 4 bytes per character.

Reply:
I agree with this assesment

Reply:
I agree that we'd better keep it as is for now. Let me merge this. We can always change the content when there is a consensus.